### ISSUE TITLE
Allow single 'top level' documentation item (uid) for overloaded methods

### ISSUE THREAD

**Issue by tobiasbunyan** [dotnet/docfx#1209]
Under Sandcastle, it was possible to insert documentation from an XML file about an overloaded method which applied to all overloads of that method:

```
<?xml version="1.0"?>
<doc>
  <members>
    <member name="EXT:O:mynamespace.myclass.myoverloadedmethod">

  blah blah blah .....
```

Under docfx, this does not currently seem to be supported.

Whilst the 'generic' uid of something like 
```
mynamespace.myclass.myoverloadedmethod*
``` 
appears throughout, it does not appear that you can use it to add content with a file override.  It's only appearance in the template appears to be used as an anchor point to get to the top of the list of overloads.

I previously 'fudged' its use by manually adding a child item to the yml file which I then defined and so could use a file override to insert content.  This of course will get broken each time the file is re-generated, so is not a viable solution.

Unless I am totally missing something in the use of file overrides?

I don't know if this needs a code change to add an item to the yml file or if it is just a simple template change (I'm guessing the former).

**Comment by vicancy** [comment]
No, batch editing overloads is not currently supported in `docfx`. You are right, overload uid `A*` is now used for cross reference only.
